src/EditorFeatures/CSharpTest/ImplementInterface/ImplementInterfaceTests.cs

```diff
@@ -8293,6 +8293,319 @@ public async Task TestInaccessibleMember_03()
             }.RunAsync();
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsImplementInterface)]
+        [WorkItem(4146, "https://github.com/dotnet/roslyn/issues/4146")]
+        public async Task TestAccessibility_Property()
+        {
+            await TestWithAllCodeStyleOptionsOffAsync(
+@"internal class Foo {}
```

Comment:
please replace all usages of Foo with Goo.  Foo flags an annoying checker that we then have to silence.